string sensorGirisAdi
int32 sensorDegeri
bool durumu
